pandas/tests/extension/base/reduce.py

```diff
@@ -25,6 +25,13 @@ class BaseNoReduceTests(BaseReduceTests):
 
     @pytest.mark.parametrize("skipna", [True, False])
     def test_reduce_series_numeric(self, data, all_numeric_reductions, skipna):
+        if isinstance(data, pd.arrays.StringArray) and all_numeric_reductions in [
```

Comment:
hmm, @TomAugspurger, @jbrockmendel  is this the pattern we are using here for skips like this?

Reply:
in indexes tests when we have a "this is tested in this other place" comment we usually return/pass instead of pytest.skip.  

It's not a perfect system, but I think of it as a way of distinguishing between "this test is skipped but would be nice to enable" vs "nothing to see here"

Reply:
Returning None rather than skipping also keeps the test output cleaner, since we print skipped tests.

Reply:
We typically refine this for a specific type by overriding this test to have custom behaviour in `extension/test_strings.py`

Reply:
ok on the test names? @jorisvandenbossche

Reply:
@dsaxton can you move this special case for string to `test_strings.py` ? (and there you an override this test method, to do the correct thing for string dtype)